lib/util.js

```diff
@@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (value.toString() === 'Invalid Date') {
```

Comment:
Insofar that it matters, the way V8 implements the Date object, getTime() is faster than valueOf() is faster than toString().  The first one is a field lookup, the others are calls into the runtime, with toString() polluting the date cache and creating a new string every time.

I'd write it as `Number.isNaN(value.getTime())` or `Object.is(value.getTime(), NaN)` to avoid keeping around an extra object, though.